media_driver/linux/common/os/mos_context_specific.cpp

```diff
@@ -335,8 +335,8 @@ MOS_STATUS OsContextSpecific::CreateSSEUIPC()
         &UserFeatureData);
     m_enableDymanicSliceShutdown = UserFeatureData.i32Data;
 
-    if (m_enableDymanicSliceShutdown < 0 && m_enableDymanicSliceShutdown < -1 ||
-        m_enableDymanicSliceShutdown > 0 && (uint32_t) m_enableDymanicSliceShutdown > m_gtSystemInfo.SliceCount)
+    if (m_enableDymanicSliceShutdown < -1 ||
```

Comment:
original code has logical error.  "m_enableDymanicSliceShutdown < 0 && m_enableDymanicSliceShutdown < -1" always equals to "m_enableDymanicSliceShutdown < -1"